internal/apps/msp/resource/deploy/handlers/configcenter/configcenter.go

```diff
@@ -106,3 +107,22 @@ func (p *provider) deleteDefaultConfigFromNacos(clusterName, addr, user, pwd, na
 	nacosClient := utils.NewNacosClient(clusterName, addr, user, pwd)
 	nacosClient.DeleteConfig(namespaceId, groupName)
 }
+
+func (p *provider) ResetAddons(info *handlers.ResourceInfo, clusterConfig map[string]string) {
+
+	if _, ok := clusterConfig[handlers.MseNacosHost]; !ok {
+		return
+	}
+	if _, ok := clusterConfig[handlers.MseNacosPort]; !ok {
+		return
+	}
+	info.Dice.AddOns = make(diceyml.AddOns)
+	info.Dice.AddOns[handlers.ResourceMSENacos] = &diceyml.AddOn{
+		Plan: "mse-nacos:basic",
+		Options: map[string]string{
+			"version":             "1.0.0",
```

Comment:
Use default version.

Reply:
can't use default version. Instead of querying the cache or database, it will just use this version